PublicMapping/districtbuilder
# Import plan first pass

## Overview

Adds the option to import a plan from a block equivalency CSV file

### Checklist


### Demo

![output gif](https://user-images.githubusercontent.com/4432106/112148471-4f8dcf80-8bb4-11eb-8525-f81e49651acd.gif)


### Notes

We have our first server test! It's an integration test that actually reaches out to S3 to download the TopoJSON files so it isn't very speedy (~15 seconds for me). I did use DE to keep resource usage low.

## Testing Instructions

- `scripts/test`
- `scripts/server`
- Export one of your existing maps to a CSV
- Import it from the home screen - the districts should remain the same

Closes #630
